ee/modules/021-cni-cilium/hooks/ee/types.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2024 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+package ee
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"sort"
+
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var hashFunc func(key string) [32]byte
+
+func init() {
+	hashFunc = newHashFunc()
+}
```

Comment:
```suggestion
var hashFunc = newHashFunc()
```

Can't you just do this? I believe this is a valid go code. You do not need the init here.